stonybook/pipeline/episode_break_prominence/add_break_prominence.py

```diff
@@ -0,0 +1,189 @@
+import os
+import lxml.etree as ET
+import pickle
+from nltk.corpus import stopwords
+from scipy import signal
+import math
+
+#parse character_coref_annotated.xml and get necessary fields
+def parse_xml(input_xml_path, output_dir):
+    parser = ET.XMLParser(huge_tree=True, remove_blank_text=True)
+    tree = ET.parse(str(input_xml_path), parser=parser)
+    root = tree.getroot()
+
+    lemmas = {}
+    para_breaks = {}
+    chapter_breaks = {}
+    sent_lemma = []
+
+    for mainchild in root:
+        chapterCounter = 0
+        if mainchild.tag == 'body':
+            for var in mainchild:
```

Comment:
Rename variable name "var" with "header". Instead of looping through the mainchild in root, you can find the body tag directly with find.

Reply:
Changed to header and now using find for 'body'